src/main/resources/wa-task-configuration-wa-wacasetype.dmn

```diff
@@ -1788,12 +1788,116 @@ else if (taskAttributes != null and taskAttributes.additionalProperties != null
       </rule>
+      <rule id="DecisionRule_1ike20b">
+        <inputEntry id="UnaryTests_0zoagxz">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1tr0yl9">
+          <text>"reconfigTaskAttributesTask"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_0hnw2hk">
+          <text>"workType"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0uzgmdb">
+          <text>if (taskAttributes != null and taskAttributes.additionalProperties != null and taskAttributes.additionalProperties.roleAssignmentId != null) then ("access_requests") else "hearing_work"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1onv4yn">
+          <text>true</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_01f8thf">
+        <inputEntry id="UnaryTests_1hvmpr3">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1xe68e0">
+          <text>"reconfigTaskAttributesTask"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_03pa0o8">
+          <text>"additionalProperties_roleAssignmentId"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0m8rp27">
+          <text>if (taskAttributes != null and taskAttributes.roleAssignmentId != null) then taskAttributes.roleAssignmentId
+else if (taskAttributes != null and taskAttributes.additionalProperties != null and taskAttributes.additionalProperties.roleAssignmentId != null) then taskAttributes.additionalProperties.roleAssignmentId else ""</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1s94e5o">
+          <text>true</text>
+        </outputEntry>
+      </rule>
+      <rule id="DecisionRule_1omyzel">
+        <inputEntry id="UnaryTests_1xwtlz2">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0jdapd9">
+          <text>"reconfigTaskAttributesTask2"</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_1ubnthy">
+          <text>"title"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_0ep1zj0">
+          <text>if (taskAttributes.lastUpdatedUser != null or taskAttributes.taskName != null or taskAttributes.dueDateTime != null or taskAttributes.caseCategory != null or taskAttributes.securityClassification != null or taskAttributes.lastReconfigurationTime != null or taskAttributes.reconfigureRequestTime != null or taskAttributes.autoAssigned != null or taskAttributes.state != null or taskAttributes.taskSystem != null or taskAttributes.indexed != null or taskAttributes.lastUpdatedTimestamp != null or taskAttributes.taskRoleResources != null or taskAttributes.executionTypeCode != null or taskAttributes.businessContext != null or taskAttributes.terminationReason != null or taskAttributes.notes != null or taskAttributes.assignmentExpiry != null or taskAttributes.workTypeResource != null) then "updatedTitle" else "Title"</text>
```

Comment:
This clause should check that if an existing internal field is accidentally re-introduced a test will fail.